### PR TITLE
Csp 3381 process rules continuation task

### PR DESCRIPTION
**Pull request by ofiriro3**
- Closes: https://github.com/elastic/security-team/issues/3381

### CODE REVIEW

File: compliance/cis_eks/rules/cis_3_2_11/data.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    ps -ef | grep kubelet
+    ```
+    The output of the above command should return something similar to `--config`
+    `/etc/kubernetes/kubelet/kubelet-config.json` which is the location of the Kubelet
+    config file.
+    Open the Kubelet config file:
+    ```
+    cat /etc/kubernetes/kubelet/kubelet-config.json
+    ```
+    Verify that `RotateKubeletServerCertificat`e argument exists and is set to `true`.
```
Comment (jeniawhite) [inline compliance/cis_eks/rules/cis_3_2_11/data.yaml:33]:
```suggestion
    Verify that `RotateKubeletServerCertificate` argument exists and is set to `true`.
```

File: compliance/cis_eks/rules/cis_3_2_11/data.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    ```
+  remediation: |
+    **Remediation Method 1:**
+    If modifying the Kubelet config file, edit the kubelet-config.json file
+    `/etc/kubernetes/kubelet/kubelet-config.json` and set the below parameter to `true`
+    ```
+    "RotateKubeletServerCertificate":true
+    ```
+    **Remediation Method 2:**
+    If using a Kubelet config file, edit the file to set RotateKubeletServerCertificate to
```
Comment (jeniawhite) [inline compliance/cis_eks/rules/cis_3_2_11/data.yaml:58]:
```suggestion
    If using a Kubelet config file, edit the file to set `RotateKubeletServerCertificate to`
```

File: compliance/cis_eks/rules/cis_3_2_11/data.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    true.
```
Comment (jeniawhite) [inline compliance/cis_eks/rules/cis_3_2_11/data.yaml:59]:
```suggestion
    `true`.
```

File: compliance/cis_eks/rules/cis_3_2_4/data.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    ```
+    The output of the above command should return something similar to `--config`
+    `/etc/kubernetes/kubelet/kubelet-config.json` which is the location of the Kubelet
+    config file.
+    Open the Kubelet config file:
+    ```
+    cat /etc/kubernetes/kubelet/kubelet-config.json
+    ```
+    Verify that the `--read-only-port argument` exists and is set to `0`.
+    If the `--read-only-port argument` is not present, check that there is a Kubelet config file
```
Comment (jeniawhite) [inline compliance/cis_eks/rules/cis_3_2_4/data.yaml:29]:
```suggestion
    If the `--read-only-port` argument is not present, check that there is a Kubelet config file
```

File: compliance/cis_eks/rules/cis_3_2_5/data.yaml [hunk 5]
```diff
@@ -0,0 +1,104 @@
+metadata:
+  id: 627afe20-4027-585e-9db2-2b5f942d769c
+  name: Ensure that the --streaming-connection-idle-timeout argument isnot set to 0 (Manual)
```
Comment (jeniawhite) [inline compliance/cis_eks/rules/cis_3_2_5/data.yaml:3]:
```suggestion
  name: Ensure that the --streaming-connection-idle-timeout argument is not set to 0 (Manual)
```

File: compliance/cis_eks/rules/cis_3_2_5/data.yaml [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+  profile_applicability: |
+    * Level 1
+  description: |
+    Do not disable timeouts on streaming connections.
+  rationale: >
+    Setting idle timeouts ensures that you are protected against
+    Denial-of-Service attacks,
+    inactive connections and running out of ephemeral ports.
+    Note: By default, `--streaming-connection-idle-timeout` is set to 4 hours which might be
```
Comment (jeniawhite) [inline compliance/cis_eks/rules/cis_3_2_5/data.yaml:12]:
```suggestion
    **Note:** By default, `--streaming-connection-idle-timeout` is set to 4 hours which might be
```

File: compliance/cis_eks/rules/cis_3_2_5/data.yaml [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+    If using the api configz endpoint consider searching for the status of
+    `"streamingConnectionIdleTimeout":"4h0m0s"` by extracting the live configuration from
+    the nodes running kubelet.
+    Set the local proxy port and the following variables and provide proxy port number and
+    node name;
+    `HOSTNAME_PORT="localhost-and-port-number"`
+    `NODE_NAME="The-Name-Of-Node-To-Extract-Configuration" from the output of`
+    `"kubectl get nodes"`
+    ```
+    kubectl proxy --port=8001 &`
```
Comment (jeniawhite) [inline compliance/cis_eks/rules/cis_3_2_5/data.yaml:44]:
```suggestion
    kubectl proxy --port=8001 &
```

**Review by jeniawhite — CHANGES_REQUESTED** [review]

File: compliance/cis_eks/rules/cis_3_2_9/data.yaml [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+    configuration changes
+    ```
+    kubectl proxy --port=8001 &
+    export HOSTNAME_PORT=localhost:8001 (example host and port number)
+    export NODE_NAME=ip-192.168.31.226.ec2.internal (example node name from
+    "kubectl get nodes")
+    curl -sSL "http://${HOSTNAME_PORT}/api/v1/nodes/${NODE_NAME}/proxy/configz"
+    ```
+    **For all three remediations:**
+    Based on your system, restart the kubelet service and check status
```
Comment (jeniawhite) [inline compliance/cis_eks/rules/cis_3_2_9/data.yaml:82]:
```suggestion
    Based on your system, restart the `kubelet` service and check status
```

**Review by jeniawhite — CHANGES_REQUESTED** [review]

**Review by jeniawhite — APPROVED** [review]

File: compliance/cis_eks/rules/cis_3_2_4/data.yaml [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+    Open the Kubelet config file:
+    ```
+    cat /etc/kubernetes/kubelet/kubelet-config.json
+    ```
+    Verify that the `--read-only-port argument` exists and is set to `0`.
+    If the `--read-only-port` argument is not present, check that there is a Kubelet config file
+    specified by `--config`. Check that if there is a `readOnlyPort` entry in the file, it is set to `0`.
+  remediation: |
+    If modifying the Kubelet config file, edit the kubelet-config.json file
+    `/etc/kubernetes/kubelet/kubelet-config.json` and set the below parameter to false
```
Comment (oren-zohar) [inline compliance/cis_eks/rules/cis_3_2_4/data.yaml:33]:
```suggestion
    `/etc/kubernetes/kubelet/kubelet-config.json` and set the below parameter to `false`
```

File: compliance/cis_eks/rules/cis_3_2_11/data.yaml [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+    ```
+  remediation: |
+    **Remediation Method 1:**
+    If modifying the Kubelet config file, edit the kubelet-config.json file
+    `/etc/kubernetes/kubelet/kubelet-config.json` and set the below parameter to `true`
+    ```
+    "RotateKubeletServerCertificate":true
+    ```
+    **Remediation Method 2:**
+    If using a Kubelet config file, edit the file to set `RotateKubeletServerCertificate to`
```
Comment (oren-zohar) [inline compliance/cis_eks/rules/cis_3_2_11/data.yaml:58]:
```suggestion
    If using a Kubelet config file, edit the file to set `RotateKubeletServerCertificate` to
```

**Review by oren-zohar — APPROVED** [review]
great work 💯

**Pull request merged**